contrib/make/test.mk

```diff
@@ -2,11 +2,13 @@
 # Tests
 #########################################################################
 
-PACKAGES_NOSIMULATION = ${shell go list ./... | grep -v simapp}
+.PHONY: test-unit
+test-unit:
+	go test ./... -short
 
-.PHONY: test-coverage
-test-coverage:
-	go test ./... $(PACKAGES_NOSIMULATION) -short \
+.PHONY: test-coverage-unit
```

Comment:
Using `$PACKAGES_NOSIMULATION` is unnecessary since simulations require a specific flag and we're already using `-short`